### PR TITLE
added helpful executemany & execute_batch examples

### PR DESCRIPTION
**Pull request by kannes**
hopefully this will make those functions more approachable for newbies. everyone tries it without proper sequences and then has to google.

### CODE REVIEW

**Comment by dvarrazzo** [comment]
I would prefer if the style of the examples was canonical (space after the commas etc) and that the single values example was shown with tuples instead of lists (because one-item tuples are a thing that tricks a lot of newcomers).

**Comment by kannes** [comment]
I'd thought about that but I did not really see a reason why tuples were preferred anyways. I will update it though.

File: doc/src/extras.rst [hunk 1]
```diff
@@ -986,6 +986,12 @@ parameters.  By reducing the number of server roundtrips the performance can be
 
 .. autofunction:: execute_batch
 
+    >>> nums = [[1],[5],[10]]
+    >>> execute_batch(cur, "INSERT INTO test (num) VALUES (%s)", nums)
+            
+    >>> tuples = ((123,"foo"), (42,"bar"), (23,"baz"))
+    >>> execute_batch(cur, "INSERT INTO test (num, data) VALUES (%s, %s)", tuples)
```
Comment (liuzheng1990) [inline doc/src/extras.rst:993]: Looks useful for a person who wants to see some simply examples.

File: doc/src/extras.rst [hunk 2]
```diff
@@ -986,6 +986,12 @@ parameters.  By reducing the number of server roundtrips the performance can be
 
 .. autofunction:: execute_batch
 
+    >>> nums = [[1],[5],[10]]
```
Comment (kannes) [inline doc/src/extras.rst:989]: Should be `nums = ((1,), (5,), (10,))`

File: doc/src/cursor.rst [hunk 3]
```diff
@@ -197,6 +197,12 @@ The ``cursor`` class
 
         Parameters are bounded to the query using the same rules described in
         the `~cursor.execute()` method.
+        
+            >>> nums = [[1],[5],[10]]
```
Comment (kannes) [inline doc/src/cursor.rst:201]: Should be `nums = ((1,), (5,), (10,))`

File: doc/src/cursor.rst [hunk 4]
```diff
@@ -197,6 +197,12 @@ The ``cursor`` class
...
+            >>> cur.executemany("INSERT INTO test (num) VALUES (%s)", nums)
+            
+            >>> tuples = ((123,"foo"), (42,"bar"), (23,"baz"))
```
Comment (kannes) [inline doc/src/cursor.rst:204]: Should be `tuples = ((123, "foo"), (42, "bar"), (23, "baz"))`

**Review by kannes — COMMENTED** [review]
I seem unable to find my repo again and I cannot make those fancy new code suggestions here for some reason.

These would be the changes necessary I think.

**Comment by kannes** [comment]
I will make a new PR with those changes.

**Comment by kannes** [comment]
Deprecated, closing in favor of https://github.com/psycopg/psycopg2/pull/1508

**Pull request closed without merging**